environments/network/ptl.tfvars

```diff
@@ -155,6 +155,24 @@ additional_routes = [
     address_prefix         = "10.141.0.0/18"
     next_hop_type          = "VirtualAppliance"
     next_hop_in_ip_address = "10.11.72.36"
+  },
+  {
+    name                   = "mgmt-nonprod"
+    address_prefix         = "10.48.0.32/27"
+    next_hop_type          = "VirtualAppliance"
+    next_hop_in_ip_address = "10.11.72.36"
```

Comment:
You only need this one, the other two will be captured in the default routes

Reply:
Ah very good point - will update